### PR TITLE
N28: FishFight

### PR DESCRIPTION
**Pull request by not-fl3**
Part of #852

### CODE REVIEW

File: content/news/028/index.md [hunk 1]
```diff
@@ -66,6 +66,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
 ## Game Updates
 
+### [Fish Fight: The Prequel][]
+
+[![fishfight](fishfight.gif)][fishfight-steam]
+
+Fish Fight is a fast-paced 2D brawler game, played by 1-4 players online
+or on a shared screen, built with [macroquad] game engine.
+
+This month it got the trailer and the game is now available to
```
Comment (17cupsofcoffee) [inline content/news/028/index.md:76]:
```suggestion
This month it got a trailer, and the game is now available to
```

File: content/news/028/index.md [hunk 2]
```diff
@@ -66,6 +66,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+wishlist on [steam][fishfight-steam]!
```
Comment (17cupsofcoffee) [inline content/news/028/index.md:77]:
```suggestion
wishlist on [Steam][fishfight-steam]!
```

File: content/news/028/index.md [hunk 3]
```diff
@@ -66,6 +66,20 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Fish Fight: The Prequel][]
```
Comment (17cupsofcoffee) [inline content/news/028/index.md:69]:
```suggestion
### [Fish Fight: The Prequel][fishfight-steam]
```

**Review by 17cupsofcoffee — CHANGES_REQUESTED** [review]
The content looks good (made a couple of tweaks but nothing major), but the GIF is over our 2.5MB limit - could you replace it with a smaller version?

I had a go at optimizing it but I could only get it down to 2.45MB - if you have any bright ideas to shrink it more, then that'd be useful, but if not feel free to use my version:

fishfight(1)

**Comment by not-fl3** [comment]
Your optimized GIF looks perfect, updated the PR! 

Thanks a lot!

**Review by 17cupsofcoffee — APPROVED** [review]

**Pull request merged**